internal/service/pipes/source_parameters.go

```diff
@@ -444,7 +444,7 @@ func sourceParametersSchema() *schema.Schema {
 									Schema: map[string]*schema.Schema{
 										"basic_auth": {
 											Type:         schema.TypeString,
-											Required:     true,
+											Optional:     true,
```

Comment:
what would setting required to false do? the same i guess?

Reply:
I have decided to stick with conventions based on the naming below